lib/cc/cli/init.rb

```diff
@@ -65,6 +64,16 @@ def exclude_paths(paths)
         expanded_paths
       end
 
+      def build_engine_config(engine_config)
+        config = { "enabled" => true }
+
+        if engine_config["default_config"]
+          config["config"] = engine_config["default_config"]
+        end
+
+        config
```

Comment:
It feels like there should be a cleaner way to do this...but I can't think of anything. Shouldn't be a blocker though.